sandbox/extension/chronometricDate_2018-03-26.xml

```diff
@@ -8,24 +8,24 @@
     dc:title="ChronometricDate"
     name="ChronometricDate" 
     namespace="http://rs.tdwg.org/dwc/terms/" 
-    rowType="http://rs.gbif.org/terms/1.0/ChronometricDate"
+    rowType="http://rs.gbif.org/terms/1.1/ChronometricDate"
     dc:issued="2017-11-30"
-    dc:modified="2018-01-24"
+    dc:modified="2018-03-26"
     dc:subject="dwc:Occurrence"
     dc:relation="http://rs.tdwg.org/dwc/terms/index.htm#ChronometricDate"
     dc:description="Extension to Occurrence to capture chronometric dating information.">
     <property name="chronometricDateID" namespace="http://rs.tdwg.org/dwc/terms/" qualName="http://rs.tdwg.org/dwc/terms/chronometricDateID" dc:description='An identifier for the set of information associated with a chronometricDate. May be a global unique identifier or an identifier specific to the data set.' examples='GUID reference to the combination of information for one ChronometricDate' type="string" required="true"/>
     <property name='reportedAge' namespace='http://rs.tdwg.org/dwc/terms/' qualName='http://rs.tdwg.org/dwc/terms/reportedAge' dc:description='The unconverted output of a dating protocol. For example, the conventional radiocarbon age.' examples='' type="string" required='false'/>
-    <property name='reportedAgeConversionProtocol' namespace='http://rs.tdwg.org/dwc/terms/' qualName='http://rs.tdwg.org/dwc/terms/reportedAgeConversionProtocol' dc:description='The method used for converting the reportedAge into a chronometric date, as captured in the maximumAge, maximumAgeUnits, minimumAge, and minimumAgeUnits fields. For example, calibration of conventional radiocarbon age.' examples='' type="string" required='false'/>
+    <property name='reportedAgeConversionProtocol' namespace='http://rs.tdwg.org/dwc/terms/' qualName='http://rs.tdwg.org/dwc/terms/reportedAgeConversionProtocol' dc:description='The method used for converting the reportedAge into a chronometric date, as captured in the maximumAge, maximumAgeReferenceSystem, minimumAge, and minimumAgeReferenceSystem fields. For example, calibration of conventional radiocarbon age.' examples='' type="string" required='false'/>
     <property name='maximumAge' namespace='http://rs.tdwg.org/dwc/terms/' qualName='http://rs.tdwg.org/dwc/terms/maximumAge' dc:description='Upper limit for the age of a specimen as determined by a dating method.' examples='' type="string" required='false'/>
-    <property name='maximumAgeUnits' namespace='http://rs.tdwg.org/dwc/terms/' qualName='http://rs.tdwg.org/dwc/terms/maximumAgeUnits' dc:description='The units associated with the maximumAge.' examples='"kya", "mya", "BP", "AD", "BCE"' type="string" required='false'/>
```

Comment:
It is not allowed to remove terms from a production extension if I remember correctly. 
This can only be done by adding a new version of the extension, i.e. a new file

Reply:
err, this file is the sandbox so all fine here.